test2.java

```diff
@@ -0,0 +1,12 @@
+class ThrowInFinally {
+    void test() {
+        try {
+        } finally {
+            throw new RuntimeException();
+            return;
+        }
+    }
+    void test2() {
+        throw new RuntimeException();
+    }
```

Comment:
RSPEC-9999: Test methods should be with format of :     nameOfMethod_stateUnderTest_expectedBehavior
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._